package.json

```diff
@@ -2,7 +2,7 @@
   "name": "trifid-handler-sparql",
   "description": "SPARQL handler for Trifid",
   "type": "module",
-  "version": "2.0.3",
```

Comment:
The version should only be changed with `npm version`.

Reply:
+1

And this should never be changed in a branch other than the `main` one.